fluentd/generate_throttle_configs.rb

```diff
@@ -17,7 +17,7 @@
   @log.debug entry
 end if @log.debug?
 
-DEFAULT_OPS_PROJECTS = ['default', 'openshift', 'openshift-infra']
+DEFAULT_OPS_PROJECTS = !ENV['OCP_OPERATIONS_PROJECTS'].nil? ? ENV['OCP_OPERATIONS_PROJECTS'].split(' ') : ['default', 'openshift', 'openshift-infra']
```

Comment:
I'm not sure if this will work since these are fluentd tag patterns, not project names.